Resources/Prototypes/_DV/GameRules/events.yml

```
@@ -166,7 +166,7 @@
   id: RatKingSpawn
   components:
   - type: StationEvent
-    weight: 7
+    weight: 5 # was 7
```

You don't need to add a comment in _DV files
```suggestion
    weight: 5
```